public/css/widget/object-grid.less

```diff
@@ -37,6 +41,6 @@ ul.item-table.group-grid {
   }
 }
 
-.content.full-width ul.item-table.group-grid {
+.content.full-width ul.item-list.object-grid {
```

Comment:
This is currently not applied, as `full-width` is not set in that case. But once the grid is not an item list anymore, this is obsolete anyway. So, please remove this rule.